sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/InsertIntoHiveTable.scala

```diff
@@ -85,6 +87,7 @@ case class InsertIntoHiveTable(
   def output: Seq[Attribute] = Seq.empty
 
   val hadoopConf = sessionState.newHadoopConf()
+  val createdTempDir = new scala.collection.mutable.ArrayBuffer[Path]
```

Comment:
use `Option`? It looks to me that we will only create one temp dir during one insertion.